views/add-a-book.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Add a New Book</title>
+  <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet">
+</head>
+<body>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+
+  <div class="container mt-5">
+    <h1 class="mb-4">Add a New Book</h1>
+    <form id="addBookForm" action="/api/books" method="POST">
+      <div class="form-group">
+        <label for="title">Title *:</label>
+        <input type="text" class="form-control" id="title" name="title" required>
+      </div>
+      
+      <div class="form-group">
+        <label for="author">Author:</label>
+        <input type="text" class="form-control" id="author" name="author">
+        <small id="authorHelpText" class="form-text text-muted">
+          Optional
+        </small>
+      </div>
+      <!-- TODO add metadata fields -->
+      <button type="submit" class="btn btn-primary">Add Book</button>
+    </form>
+  </div>
+  
+  <script>
+    const form = document.getElementById("addBookForm");
+
+    const serialize_form = form => JSON.stringify(
+      Array.from(new FormData(form).entries())
+        .reduce((m, [ key, value ]) => Object.assign(m, { [key]: value }), {})
+    );
```

Comment:
I took this from StackOverflow- not changing it in this PR, but might change it if I get inspired to use Vue or something.